File: ruby/splitwise/lib/splitwise/models/group.rb
module Splitwise
  module Models
    class Group
      attr_accessor :id, :image_url, :name, :users

      def initialize(id:, name:, image_url:, users:)
        self.id = id
        self.name = name
        self.image_url = image_url
        self.users = users
      end
    end
  end
end

File: java/splitwise/src/main/java/gagan93jtg/lld/models/Group.java
package gagan93jtg.lld.models;

import java.util.ArrayList;

public class Group {
  int id;
  String name;
  ArrayList<User> users;

  public Group(int id, String name, ArrayList<User> users) {
    this.id = id;
    this.name = name;
    this.users = users;
  }
}
